2021-09-20 22:28:00,768 INFO    MainThread:540428 [wandb_setup.py:_flush():69] setting env: {}
2021-09-20 22:28:00,768 INFO    MainThread:540428 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-20 22:28:00,768 INFO    MainThread:540428 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-20 22:28:00,768 INFO    MainThread:540428 [wandb_init.py:_log_setup():342] Logging user logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_222800-1h7dip52/logs/debug.log
2021-09-20 22:28:00,768 INFO    MainThread:540428 [wandb_init.py:_log_setup():343] Logging internal logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_222800-1h7dip52/logs/debug-internal.log
2021-09-20 22:28:00,769 INFO    MainThread:540428 [wandb_init.py:init():375] calling init triggers
2021-09-20 22:28:00,769 INFO    MainThread:540428 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'NOW': '2021-09-20T22:28:00', 'ARCHITECTURE': 'single-layer-perceptron', 'DATASET': 'MNIST-handwrite', 'DATA_PATH': '../dataset/', 'BATCH_SIZE': 256, 'NUM_WORKERS': 1, 'EPOCHS': 10, 'LEARNING_RATE': 0.0001, 'SAVEFIG_NAME': 'result.png'}
2021-09-20 22:28:00,769 INFO    MainThread:540428 [wandb_init.py:init():424] starting backend
2021-09-20 22:28:00,769 INFO    MainThread:540428 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-20 22:28:00,777 INFO    MainThread:540428 [backend.py:ensure_launched():135] starting backend process...
2021-09-20 22:28:00,781 INFO    MainThread:540428 [backend.py:ensure_launched():140] started backend process with pid: 540471
2021-09-20 22:28:00,781 INFO    MainThread:540428 [wandb_init.py:init():429] backend started and connected
2021-09-20 22:28:00,783 INFO    MainThread:540428 [wandb_init.py:init():477] updated telemetry
2021-09-20 22:28:00,783 INFO    MainThread:540428 [wandb_init.py:init():500] communicating current version
2021-09-20 22:28:01,259 INFO    MainThread:540428 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-20 22:28:01,259 INFO    MainThread:540428 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-20 22:28:01,563 INFO    MainThread:540428 [wandb_init.py:init():540] starting run threads in backend
2021-09-20 22:28:02,430 INFO    MainThread:540428 [wandb_run.py:_console_start():1601] atexit reg
2021-09-20 22:28:02,430 INFO    MainThread:540428 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-09-20 22:28:02,431 INFO    MainThread:540428 [wandb_run.py:_redirect():1480] Redirecting console.
2021-09-20 22:28:02,432 INFO    MainThread:540428 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-20 22:28:02,432 INFO    MainThread:540428 [wandb_init.py:init():565] run started, returning control to user process
2021-09-20 22:28:02,432 INFO    MainThread:540428 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-09-20 22:28:02,433 INFO    MainThread:540428 [wandb_run.py:_restore():1543] restore
2021-09-20 22:28:04,947 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 735
}

2021-09-20 22:28:05,339 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 735
}

2021-09-20 22:28:05,580 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 5574
}

2021-09-20 22:28:05,681 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 5574
}

2021-09-20 22:28:05,782 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 5574
}

2021-09-20 22:28:05,885 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 5574
}

2021-09-20 22:28:05,986 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1572
  total_bytes: 5574
}

2021-09-20 22:28:06,089 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1598
  total_bytes: 5574
}

2021-09-20 22:28:06,192 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1598
  total_bytes: 5574
}

2021-09-20 22:28:06,295 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5574
  total_bytes: 5574
}

2021-09-20 22:28:06,397 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5574
  total_bytes: 5574
}

2021-09-20 22:28:06,499 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5574
  total_bytes: 5574
}

2021-09-20 22:28:06,602 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5574
  total_bytes: 5574
}

2021-09-20 22:28:06,705 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5574
  total_bytes: 5574
}

2021-09-20 22:28:06,997 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5574
  total_bytes: 5574
}

2021-09-20 22:28:07,100 INFO    MainThread:540428 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5574
  total_bytes: 5574
}

2021-09-20 22:28:08,304 INFO    MainThread:540428 [wandb_run.py:_show_files():1915] logging synced files
